notification/events.go

```diff
@@ -111,11 +152,24 @@ func (t *notificationHandler) addNotificationEvent(ctx context.Context, event mo
 			continue
 		}
 
+		// TODO: How does this get unset ?
```

Comment:
@moshloop @adityathebe We add this error to a notification but we where do we unset it, because it gets skipped when it is set

Reply:
there's a postgres trigger

```
reset_notification_error_before_update_trigger BEFORE UPDATE ON notifications FOR EACH ROW EXECUTE FUNCTION reset_notification_error_before_update()
```

Reply:
We should probably just update the comment here with that detail